Library/Formula/libxmp-lite.rb

```diff
@@ -0,0 +1,25 @@
+class LibxmpLite < Formula
+  homepage "http://xmp.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/xmp/libxmp/4.3.5/libxmp-lite-4.3.5.tar.gz"
+  sha1 "45ae99bf939f1fe2cb8c75ec76406cbb028a1bf8"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.c").write <<-'EOS'.undent
+      #include <stdio.h>
+      #include <libxmp-lite/xmp.h>
+
+      int main(int argc, char** argv){
+        printf("libxmp-lite %s/%c%u\n", XMP_VERSION, *xmp_version, xmp_vercode);
```

Comment:
Ah, gotcha. The newline was it; I was trying to spot the thing that was going to be interpolated. :+1: